assembl/graphql/view.py

```diff
@@ -13,9 +13,15 @@
 class LoggingMiddleware(object):
     def resolve(self, next, source, gargs, context, info, *args, **kwargs):
         if source is None:
+            variables = {}
+            for key, value in info.variable_values.items():
+                if 'password' in key or 'Password' in key:
+                    variables[key] = 'xxxxxxxxxxxxx'
```

Comment:
Is it safe to override the info key in the middleware? Would it be possible for logging middleware to affect the info fields downstream for the mutation?

Reply:
I don't override anything, it's a new variables dict here, only used for the log.